src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -567,11 +569,11 @@ public void renderStructure(@NotNull final BlockPos startingPos, @NotNull final
                 fakeWorld.setTileEntity(aBlockList.pos, tileentity);
             }
 
-            for (final Template.BlockInfo aBlockList : blockList)
+            for (final Template.BlockInfo blockInfo : blockList)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this loop with the one that starts on line 558. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS3047)